docker/Dockerfile

```diff
@@ -0,0 +1,74 @@
+FROM ubuntu:16.04
```

Comment:
Ubuntu over alpine because this is intended to be a development environment and not a deploy one.

Going to polish a bit the commands to make it smaller.